stock_shipment_composer/models/stock_shipment_composer_line.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class StockShipmentComposerLine(models.Model):
+    _name = "stock.shipment.composer.line"
+    _description = "Stock Shipment Composer Line"
+
+    composer_id = fields.Many2one(
+        "stock.shipment.composer", required=True, ondelete="cascade"
+    )
+    move_id = fields.Many2one("stock.move", required=True)
+    picking_id = fields.Many2one(related="move_id.picking_id", string="Transfer")
+    quantity = fields.Float(required=True)
+    product_uom_qty = fields.Float(related="move_id.product_uom_qty")
+    reserved_availability = fields.Float(related="move_id.reserved_availability")
+    quantity_done = fields.Float(related="move_id.quantity_done")
+    uom_id = fields.Many2one(related="move_id.product_uom", string="UoM")
+    move_state = fields.Selection(related="move_id.state", string="Move Status")
+    remarks = fields.Text()
+    state = fields.Selection(related="composer_id.state", store=True)
+    reserved_enough = fields.Boolean(compute="_compute_reserved_enough")
```

Comment:
Can we add location fields.
```suggestion
    reserved_enough = fields.Boolean(compute="_compute_reserved_enough")
    location_id = fields.Many2one(
        related="move_id.location_id", string="Source Location"
    )
    location_dest_id = fields.Many2one(
        related="move_id.location_dest_id", string="Destination Location"
    )
```